src/analysis/completion.mli

```diff
@@ -47,6 +47,10 @@ val raw_info_printer : raw_info ->
   | `Concat of string * Extend_protocol.Reader.outcometree
   ]
 
+type keywords =
+  | Exclude
+  | Suggest of string list
```

Comment:
Is there a good reason to pass a list of keywords as argument?
It seems we're always passing the same list (all the keywords), so perhaps the function should just be parametrized by `suggest_keywords:bool`?

Reply:
No longer present in the up to date patch